metadetect/metadetect.py

```diff
@@ -72,22 +72,13 @@ def _set_ormask(self):
 
             if band == 0:
                 ormask = obs.ormask.copy()
+                bmask = obs.bmask.copy()
```

Comment:
why is the bmask being "ored" now?

Reply:
I thought it would be good to save the or'ed bmask across the images used as well.